master/static/srv/insert_experiment_group.sql

```diff
@@ -0,0 +1,11 @@
+WITH g AS (
+  INSERT INTO experiment_groups (name, project_id)
+  VALUES ($1, $2)
+  RETURNING id, name, project_id
+), ge AS (
```

Comment:
In `insert_experiment_group.sql` as I understand it, I think there's no need to then check the count; you could select `SELECT g.id, g.name, g.project_id, 0 AS num_experiments`